src/template.ts

```diff
@@ -129,15 +129,17 @@ function getCss(theme: string, fontSize: string) {
     }`
 }
 
-export function getHtml(parsedReq: ParsedRequest, course: any) {
-  const { theme, md, fontSize, widths, heights } = parsedReq
-  const { square_cover_large_url, title, instructor } = course
-  const images = [square_cover_large_url]
-  const text = title
-  const adjustedFontSize =
-    text.length > 60 ? (text.length > 80 ? "52px" : "56px") : fontSize
-
-  return `<!DOCTYPE html>
+export function getHtml(parsedReq: ParsedRequest, resource: any) {
+    const { theme, md, fontSize, widths, heights, resourceType } = parsedReq
+    // TODO: this should be able to handle any Resource (ContentModel)
```

Comment:
I didn't take it past this point, but from here we can switch on resourceType and make cards however we like.